### DISCUSSION TITLE
Not able to get ltex-ls working

### DISCUSSION THREAD

**Discussion by DaitiDay** [mason-org/mason.nvim#1510] [Q&A]
As for the title, I installed the ltex ls using mason but once I open a tex file I get a `Client 1 quit with exit code 1 and signal 0`. Checking with `:LspInfo` it evertything seems fine (meaning ltex is in the list of the configured servers and the cmd is executable) but `0 client(s) attached to this buffer`. There's nothing relevant in `:MasonLog`, except the fact that `Installation succeeded for Package(name=ltex-ls)`. 
I apologize if it's a stupid mistake I'm making, it's my first time experimenting with LSP and similar.

**Comment by williamboman** [comment]
`ltex-ls` has a dependency on Java 11+. As of writing you'll have to ensure `java` is in your PATH. Maybe that's what's causing issues? `:LspLog` might be helpful.

**Comment by henry-hsieh** [comment]
Hi @williamboman,
`ltex-ls` has platform dependent [tarball](https://github.com/valentjn/ltex-ls/releases/tag/16.0.0). The [installation guide](https://valentjn.github.io/ltex/ltex-ls/installation.html#installation) also mentions the platform dependent installation does not require Java. Can mason switch to the platform dependent installation in case the installation of Java is unavailable for some users.

**Reply by henry-hsieh** [reply]
Never mind. I saw this [issue](https://github.com/williamboman/mason.nvim/issues/1531) has been raised several times.
